group-aws/README.md

```diff
@@ -9,12 +9,25 @@ There are currently no regular meetings held. The IRC channel is the
 main and official place to contact the members. For specific issues and
```

Comment:
That's already in the Contacts at the bottom, but could make it more explicit here too.